airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RefreshSchemaActivityImpl.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.config.ActorCatalogFetchEvent;
+import io.airbyte.config.persistence.ConfigRepository;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+public class RefreshSchemaActivityImpl implements RefreshSchemaActivity {
+
+  private final Optional<ConfigRepository> configRepository;
+
+  public RefreshSchemaActivityImpl(Optional<ConfigRepository> configRepository) {
+    this.configRepository = configRepository;
+  }
+
+  @Override
```

Comment:
Unrelated to the ticket for this PR, but could we also add the tracing annotations/tags to this activity so that it shows up in DataDog?  Take a look at the other activities for an example.  If you need to add a tag key, please wait until [this PR](https://github.com/airbytehq/airbyte/pull/18381) is merged and add those tags to the `ApmTraceConstants` class.  Totally fine if you add all of this in a follow-on PR.

Reply:
could you explain this a bit more? is each new activity considered a different 'span'?